tests/main/views/test_direct_award.py

```diff
@@ -24,6 +29,18 @@ def setup(self):
     def teardown(self):
         super(DirectAwardSetupAndTeardown, self).teardown()
 
+    def _create_project_data(self):
+        return {
+            'project': {
+                'userId': self.user_id,
+                'name': self.direct_award_project_name,
+            },
+            'updated_by': str(self.user_id)
+        }.copy()
+
+    def _updated_by_data(self):
+        return {'updated_by': str(self.user_id)}.copy()
```

Comment:
Why do we need `copy` here? A new dict will get created on every call anyway... won't it?

Reply:
Looks like you're right.